azure/AzureDataBricksJob/src/main/java/com/microsoft/pnp/GeoFinder.java

```diff
@@ -71,26 +68,18 @@ public static GeoFinder createGeoFinder(URL shapeFileUrl) throws IOException {
             String typeName = typeNames[0];
 
             logger.info(String.format("Reading content %s", typeName));
-            ReprojectingFeatureCollection features = new ReprojectingFeatureCollection(
-                    dataStore.getFeatureSource(typeName).getFeatures(),
-                    CRS.decode("epsg:4269"),
-                    CRS.decode("epsg:4326")
-            );
             FeatureSource featureSource = new SpatialIndexFeatureSource(
-                    new SpatialIndexFeatureCollection(features));
+                    new SpatialIndexFeatureCollection(dataStore.getFeatureSource(typeName).getFeatures()));
 
             FilterFactory2 filterFactory = CommonFactoryFinder.getFilterFactory2();
             PropertyName propertyName = filterFactory.property(dataStore
                     .getSchema(typeName)
                     .getGeometryDescriptor()
                     .getName());
             return new GeoFinder(featureSource, filterFactory, propertyName);
-        } catch (IOException ioe) {
-            logger.error(String.format("Error loading Geospatial data from %s", shapeFileUrl), ioe);
-            throw ioe;
-        } catch (FactoryException fe) {
-            logger.error(String.format("Error loading Geographic Coordinate System"), fe);
-            throw new IOException(fe);
+        } catch (IOException ex) {
+            logger.error(String.format("Error loading Geospatial data from %s", shapeFileUrl));
```

Comment:
Nit: do we want to capture the exception still here in this log call like the following.  (I might be out of my context here, so no big deal if that doesn't make sense)
```suggestion
            logger.error(String.format("Error loading Geospatial data from %s", shapeFileUrl), ex);
```

Reply:
That would provide more data in the failure instance, I'll run a quick test and include the change assuming it all works as expected.

Reply:
Confirmed that it builds and runs.  I didn't have any exceptions so I didn't get to see it in action, but should provide better output in the case of an error.